### PR TITLE
Use pg short code for PostgresServer

### PR DESCRIPTION
**Pull request by byucesoy**
Currently pg is used for Page model, but it is an internal model and doesn't need a very good representation compared to PostgresServer, which is much more visible and the primary model of a key service. We decided to bite the bullet and make pg the short code for PostgresServer while it is still early.

### CODE REVIEW

File: migrate/20230915_postgres.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    create_table(:postgres_server) do
+      column :id, :uuid, primary_key: true, default: nil
+      column :created_at, :timestamptz, null: false, default: Sequel.lit("now()")
+      column :updated_at, :timestamptz, null: false, default: Sequel.lit("now()")
+      column :project_id, :project, type: :uuid, null: false
```
Comment (enescakir) [inline migrate/20230915_postgres.rb:9]: Vm doesn't have `project_id`. The association is established via tags. Should we do same thing here? Or make this thing to vm?

For authorizing users to access resources, we need tags anyway.

**Review by fdr — APPROVED** [review]
Can you think of any conflicts for "pa" ? I can't right now.

File: migrate/20230915_postgres.rb (hunk 1 above)
Comment (byucesoy) [reply migrate/20230915_postgres.rb]: Resolving this now. We can continue this discussion in main postgres PR.

**Comment by byucesoy** [comment]
> Can you think of any conflicts for "pa" ? I can't right now.
I cannot as well.

**Pull request merged**